netbox_agent/logging.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+
+def init_log():
+    logger = logging.getLogger('netbox_agent')
+    logger.setLevel(logging.DEBUG)
+
+    logger_formatter = logging.Formatter('%(name)s - %(levelname)s - %(message)s')
+
+    fh = logging.FileHandler('netbox.log')
```

Comment:
Do we really need this? It seems to me that it makes sense for daemons but not much here, and I'm not sure of what would happen on a read-only filesystem or other weird scenarios.

Reply:
I've simplified everything